torch/_torch_docs.py

```diff
@@ -8294,8 +8294,8 @@ def merge_dicts(*dicts):
           then the singular values of each matrix in the batch is returned in descending order.
 
 .. note:: The implementation of SVD on CPU uses the LAPACK routine `?gesdd` (a divide-and-conquer
-          algorithm) instead of `?gesvd` for speed. Analogously, the SVD on GPU uses the MAGMA routine
-          `gesdd` as well.
+          algorithm) instead of `?gesvd` for speed. Analogously, the SVD on GPU uses the cuSOLVER routines
```

Comment:
This is not totally accurate because we fallback to MAGMA if cuSOLVER is not enabled.

Maybe something like,

"SVD on GPU uses the cuSOLVER routines ... on CUDA XX.X and later, and the MAGMA routines ... on earlier versions of CUDA."

Reply:
Fixed